frontend/src/routes/Applications/ApplicationsPage.tsx

```diff
@@ -1,37 +1,64 @@
 import { discoveredApplicationsState, discoveredOCPAppResourcesState } from '../../atoms'
 import { useTranslation } from '../../lib/acm-i18next'
 import { queryRemoteArgoApps, queryOCPAppResources } from '../../lib/search'
 import { useQuery } from '../../lib/useQuery'
 import { NavigationPath } from '../../NavigationPath'
+import { LoadingPage } from '../../components/LoadingPage'
 
 const ApplicationsOverviewPage = lazy(() => import('./Overview'))
 const AdvancedConfigurationPage = lazy(() => import('./AdvancedConfiguration'))
 
 export default function ApplicationsPage() {
-    const location = useLocation()
     const { t } = useTranslation()
+    const location = useLocation()
+    const applicationsMatch = useRouteMatch()
+    const advancedMatch = matchPath(location.pathname, NavigationPath.advancedConfiguration)
 
-    const { data, startPolling } = useQuery(queryRemoteArgoApps)
-    const dataOCPResources = useQuery(queryOCPAppResources).data
-    const startPollingOCPResources = useQuery(queryOCPAppResources).startPolling
+    const { data, loading, startPolling } = useQuery(queryRemoteArgoApps)
+    const {
+        data: dataOCPResources,
+        loading: loadingOCPResources,
+        startPolling: startPollingOCPResources,
+    } = useQuery(queryOCPAppResources)
+    const [timedOut, setTimedOut] = useState<boolean>()
+    const setDiscoveredApplications = useSetRecoilState(discoveredApplicationsState)
+    const setDiscoveredOCPAppResources = useSetRecoilState(discoveredOCPAppResourcesState)
 
-    useEffect(startPolling, [startPolling])
-    useEffect(startPollingOCPResources, [startPollingOCPResources])
+    useEffect(() => {
+        if (applicationsMatch.isExact) {
+            // No need to poll for Advanced configuration page
+            startPolling()
+            startPollingOCPResources()
+        }
+    }, [applicationsMatch, startPolling, startPollingOCPResources])
 
-    const setDiscoveredAppilcations = useSetRecoilState(discoveredApplicationsState)
-    const setDiscoveredOCPAppResources = useSetRecoilState(discoveredOCPAppResourcesState)
     useEffect(() => {
         const remoteArgoApps = data?.[0]?.data?.searchResult?.[0]?.items || []
-        setDiscoveredAppilcations(remoteArgoApps)
+        setDiscoveredApplications(remoteArgoApps)
         const ocpAppResources = dataOCPResources?.[0]?.data?.searchResult?.[0]?.items || []
         setDiscoveredOCPAppResources(ocpAppResources)
-    }, [data, dataOCPResources, setDiscoveredAppilcations, setDiscoveredOCPAppResources])
+    }, [data, dataOCPResources, setDiscoveredApplications, setDiscoveredOCPAppResources])
+
+    // failsafe in case search api is sleeping
+    useEffect(() => {
+        const handle = setTimeout(() => {
+            setTimedOut(true)
+        }, 5000)
+
+        return () => {
+            clearInterval(handle)
+        }
+    }, [])
+
+    if ((loading || loadingOCPResources) && !timedOut) {
+        return <LoadingPage />
```

Comment:
<LoadingPage> will align with the top-level <LoadingPage> that we render when waiting for recoil data.

<AcmLoadingPage> uses a different alignment which resulted in a visual jump.